bootstrap.sh

```diff
@@ -263,9 +263,8 @@ add_option HTTP_CURL_ENABLED ${TRUE} "DISABLE_CURL"
 add_option LIBARCHIVE_ENABLED ${TRUE} "DISABLE_LIBARCHIVE"
 add_dependency LIBARCHIVE_ENABLED "libarchive"
 
-add_option EXECUTE_SCRIPT_ENABLED ${FALSE} "ENABLE_SCRIPTING"
-add_dependency EXECUTE_SCRIPT_ENABLED "python"
-add_dependency EXECUTE_SCRIPT_ENABLED "lua"
+add_option PYTHON_SCRIPTING_ENABLED ${FALSE} "ENABLE_PYTHON_SCRIPTING"
+add_option LUA_SCRIPTING_ENABLED ${FALSE} "ENABLE_LUA_SCRIPTING"
```

Comment:
I imagined the ExecuteScript + script executors architecture with 3 extensions:

- scripting: ExecuteScript without any executors
- python: PythonScriptExecutor and support for python processors
- lua: LuaScriptExecutor and lua processors

How does it work now?